pkg/simple/client/notification/options.go

```diff
@@ -18,11 +18,14 @@ package notification
 
 type Options struct {
 	Endpoint string `json:"endpoint,omitempty" yaml:"endpoint,omitempty"`
+
+	Enable bool `json:"enable" yaml:"enable"`
```

Comment:
```suggestion
```

Reply:
delete enable ?

Reply:
If enable is removed, how can the user modify it through the configuration file? @wansir

Reply:
If the user has configured the endpoint, then notification is enabled?

Reply:
yeap

Reply:
It's similar to Auditing and Events.